pkg_defs/ska3-template/build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+mkdir ${PREFIX}/bin
```

Comment:
This should probably be `mkdir -p` on unix.   Do we need ska3-template or a skare launcher on windows?

Reply:
Not to my knowledge.  If we find a need then we'll address it.

With regards to the `-p`, hasn't this been working?  Don't we assume from the build process that `${PREFIX}` must exist already?

Reply:
The "mkdir" fix is here because this recipe it wasn't working for me on conda 4.7 .  ${PREFIX} appeared to exist, but 'bin' did not in that case I think.  I was just thinking "-p" might be more future-proof but probably don't have to worry about that

Reply:
I am just worried that `mkdir -p` can make any arbitrarily path and do unexpected things.  If `$PREFIX` exists then `mkdir $PREFIX/bin` should never fail, no?